pages/common/diff.md

```diff
@@ -26,3 +26,7 @@
 - Compare directories, only showing the names of files that differ:
 
 `diff -rq {{old_directory}} {{new_directory}}`
+
+- Create patch file from the differences of two directories or files:
+
+`diff -Naur {{old}} {{new}} > differences.patch`
```

Comment:
I'm not a fan of too many short arguments in this way, also I think a patch file could be generated in any of these examples as well.

Reply:
I'm not a big fan either, and it is also partly against tldr's guideline, but every other command in the list had it and I followed their steps.

Regarding your last point though, the simple answer is no it cannot, because the following are typically expected in a conventional patch file:
1. `-a, --text` which will treat all files as text
2.  `-N, --new-file` which will treat absent files as empty
3.  `-u, --unified` which brings in the 3 line context to the diff

Reply:
I will be more than happy to change all the arguments to the long form as that is my preference as well.

Reply:
I agree that we should use long flags everywhere, so please go ahead.
What do you think about this descripption:
> Create a patch file from the difference of two files, treating them as text files and treating absent files as empty:
>
> \`diff --new-file --text {{path/to/file1}} {{path/to/file2}} > {{path/to/diff.patch}}\`

Could need some improvent tough. I left out the `-r`  and `-u `flag, since both were covered by other examples and this one is already quite long.